### PR TITLE
fix: 🐛 Fix ghost shadow on thumb

### LINKED ISSUE #1087: Fix trailing ghost image for thumbnail drag-n-drop
### Describe the Bug

Thumbnail dragging creates a ghost image that sometimes lags behind the primary drag image. See this GIF:

wbd4urWgBH

### What steps can we follow to reproduce the bug?

1. Select a thumbnail to drag
2. Drag rapidly
3. Enjoy mouse cursor ghost like it's the late 90s

### Context

I recommend resolving this by:

- Updating the react-dnd library to latest
- Leveraging it's new `useDrag` and `useDrop` hooks
- Removing `DragPreview` entirely
    - This is already provided for us
    - Verify that we don't need to conditionally have this added for `touch` drag and drop

I played with an implementation here that removed the ghost image and made things more performant. You can take a peak at the implementation as it may be helpful

- [`useDrag`](https://github.com/OHIF/Viewers/pull/946/files#diff-fc96aa3855d4c1af1168f98809c0ee9d)
- [`useDrop`](https://github.com/OHIF/Viewers/pull/946/files#diff-c316ae52231a93efe72524bab3ea705f)

### PR DESCRIPTION
**Pull request by ladeirarodolfo**
Updated react-dnd package for the lates. Refactored thumbnail component
### Includes
- Minor refactoring implementation of thumbnail (Danny suggestion [useDrag](https://github.com/OHIF/Viewers/pull/946/files#diff-fc96aa3855d4c1af1168f98809c0ee9d))
- Updated react-dnd package version
- Removed unnecessary file implementation (use of ThumbnailEntry and related ones)

### Screenshoot
Oct-28_-2019-3_28-PM
### Closes 
#1087

### CODE REVIEW

File: platform/ui/src/components/studyBrowser/StudyBrowser.js [hunk 1]
```diff
@@ -1,62 +1,102 @@
...
+                    // Study
+                    studyInstanceUid={studyInstanceUid} // used by drop
+                    // Thumb
+                    altImageText={altImageText}
+                    imageId={imageId}
+                    instanceNumber={instanceNumber}
+                    displaySetInstanceUid={displaySetInstanceUid} // used by drop
+                    numImageFrames={numImageFrames}
+                    seriesDescription={seriesDescription}
+                    seriesNumber={seriesNumber}
```
Comment (dannyrb) [inline platform/ui/src/components/studyBrowser/StudyBrowser.js]: You nixed `data-cy=thumbnail-list` from the thumbnail container. This is causing E2E test failures.

File: platform/ui/src/components/studyBrowser/Thumbnail.js [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+import './Thumbnail.styl';
+
+function ThumbnailFooter({ seriesDescription, seriesNumber, instanceNumber, numImageFrames }) {
+  const infoOnly = !seriesDescription;
+
+  return (
+    <div className={classNames('series-details', { 'info-only': infoOnly })}>
+      <div className="series-description">{seriesDescription}</div>
+      <div className="series-information">
```
Comment (dannyrb) [inline platform/ui/src/components/studyBrowser/Thumbnail.js]: `series-information` looks to contain 3 values. Can use flexbox and `justify-content` to align these properly?

Also, if all three do not have values, can we conditionally render `series-information`? (This is common w/ `SR`)

File: platform/ui/src/components/studyBrowser/Thumbnail.styl [hunk 3]
```diff
@@ -62,33 +67,36 @@
...
 
   .series-information
-    display: flex
-    flex-direction: row
-    align-items: center
-
-    .item
-      flex: 1
-      text-align: center
+    padding-right: 4px
```
Comment (dannyrb) [inline platform/ui/src/components/studyBrowser/Thumbnail.styl]: Prefer `flex` over floats

File: platform/ui/src/components/studyBrowser/Thumbnail.js (hunk 2 above)
Comment (ladeirarodolfo) [reply platform/ui/src/components/studyBrowser/Thumbnail.js]: ok

**Pull request merged**